src/MainWindow.vala

```diff
@@ -170,7 +173,18 @@ public class Sideload.MainWindow : Hdy.ApplicationWindow {
                 notification.set_body (_("The app was installed"));
             }
 
+            var icon = get_application_icon ();
+            if (icon != null) {
+                notification.set_icon (new ThemedIcon (icon));
+            }
             application.send_notification ("installed", notification);
         }
     }
+
+    private string? get_application_icon () {
+        var desktop_file_path = (string)GLib.Environment.get_home_dir () + "/.local/share/flatpak/exports/share/applications/" + app_id + ".desktop";
+        var desktop_info = new GLib.DesktopAppInfo.from_filename (desktop_file_path);
```

Comment:
Instead of hardcoding a file path the better way to do this would be:
```suggestion
        var desktop_info = new GLib.DesktopAppInfo (app_id + ".desktop");
```